src/tasks/AutoBidAuctionTask.py

```
@@ -0,0 +1,406 @@
+        self._check_stop()
+        box_exception_confirm = self.box_of_screen_scaled(1920, 1080, 1111, 692, width_original=105, height_original=43)
+        self.operate_click(box_exception_confirm, after_sleep=0.3)
+        self.log_info("已点击异常确认框")
+
+    def _stage_match(self, box_match, box_confirm, box_bid, re_match, re_confirm, re_bid):
+        while True:
+            self._check_stop()
+            self.log_info("等待匹配开始...")
+
+            if self.ocr(box=box_bid, match=re_bid):
+                self.log_info("检测到已在出价界面，跳过匹配步骤")
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)